workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen/EntryPoint.cs

```diff
@@ -27,6 +31,8 @@ public static int Main(string[] args)
             }
             catch (Exception e)
             {
+                logger.Error(e, "Code generation failed due to exception");
```

Comment:
```suggestion
                logger?.Error(e, "Code generation failed due to exception");
```

If this throws:

```csharp
var options = CodeGeneratorOptions.ParseArguments(args);
```

then `logger` will be null